selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/GitPanelTest.java

```diff
@@ -0,0 +1,421 @@
+import com.google.inject.name.Named;
+import java.net.URL;
+import java.nio.file.Paths;
+import org.eclipse.che.selenium.core.SeleniumWebDriver;
+import org.eclipse.che.selenium.core.client.TestProjectServiceClient;
+import org.eclipse.che.selenium.core.client.TestUserPreferencesServiceClient;
+import org.eclipse.che.selenium.core.constant.TestMenuCommandsConstants;
+import org.eclipse.che.selenium.core.project.ProjectTemplates;
+import org.eclipse.che.selenium.core.user.TestUser;
+import org.eclipse.che.selenium.core.workspace.TestWorkspace;
+import org.eclipse.che.selenium.pageobject.AskDialog;
+import org.eclipse.che.selenium.pageobject.AskForValueDialog;
+import org.eclipse.che.selenium.pageobject.CodenvyEditor;
+import org.eclipse.che.selenium.pageobject.Ide;
+import org.eclipse.che.selenium.pageobject.Loader;
+import org.eclipse.che.selenium.pageobject.Menu;
+import org.eclipse.che.selenium.pageobject.ProjectExplorer;
+import org.eclipse.che.selenium.pageobject.Wizard;
+import org.eclipse.che.selenium.pageobject.git.GitCompare;
+import org.eclipse.che.selenium.pageobject.git.GitPanel;
+import org.eclipse.che.selenium.pageobject.machineperspective.MachineTerminal;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/** @author Mykola Morhun */
+public class GitPanelTest {
+
+  private static final String JAVA_PLAIN_NON_GIT_PROJECT_NAME = "non-git-java-project";
+  private static final String NODE_JS_GIT_PROJECT_NAME = "node-js";
+  private static final String JAVA_SPRING_GIT_PROJECT_NAME = "web-java-spring";
+  private static final String NEW_PROJECT_NAME = "new-project";
+  private static final String RENAMED_JAVA_SPRING_GIT_PROJECT_NAME = "java-spring";
+
+  private static final String NODE_JS_EDITED_FILE_NAME = "app.js";
+  private static final String JAVA_SPRING_EDITED_FILE1_NAME = "AppController.java";
+  private static final String JAVA_SPRING_EDITED_FILE2_NAME = "web.xml";
+  private static final String JAVA_SPRING_DELETED_FILE_NAME = "spring-servlet.xml";
+  private static final String JAVA_SPRING_ADDED_FILE_NAME = "test.java";
+
+  @Inject
+  @Named("github.username")
+  private String gitHubUsername;
+
+  @Inject private SeleniumWebDriver seleniumWebDriver;
+  @Inject private TestUser productUser;
+  @Inject private TestWorkspace workspace;
+  @Inject private Ide ide;
+  @Inject private Loader loader;
+  @Inject private TestProjectServiceClient projectServiceClient;
+  @Inject private Wizard wizard;
+  @Inject private Menu menu;
+  @Inject private ProjectExplorer projectExplorer;
+  @Inject private CodenvyEditor editor;
+  @Inject private MachineTerminal terminal;
+  @Inject private AskDialog askDialog;
+  @Inject private AskForValueDialog askForValueDialog;
+  @Inject private TestUserPreferencesServiceClient testUserPreferencesServiceClient;
+  @Inject private org.eclipse.che.selenium.pageobject.git.Git git;
```

Comment:
Add **org.eclipse.che.selenium.pageobject.git.Git** into the import section.

Reply:
thank you